app/controllers/users/profiles_controller.rb

```diff
@@ -6,6 +6,17 @@ class ProfilesController < ApplicationController
     before_action :set_profile
     before_action :authorize_profile!
 
+    def edit; end
```

Comment:
let's add facade PORO.

https://medium.com/kkempin/facade-design-pattern-in-ruby-on-rails-710aa88326f

So, you will have `@facade = Profiles::EditFacade.new(profile: @profile)` in your controller.
All variables and info which you need in view will be in facade. This will reduce the controller logic (https://blog.appsignal.com/2020/03/18/facade-pattern-in-rails-for-performance-and-maintainability.html -- good example in controllers and views benefits part).